src/server_manager/web_app/app.ts

```diff
@@ -602,6 +602,7 @@ export class App {
     } else {
       await this.showDigitalOceanCreateServer(this.digitalOceanAccount);
     }
+    bringToFront();
```

Comment:
Why is this?

Reply:
This is returning from the OAuth flow, which leaves the user looking at a blank tab in their main browser.

Reply:
Interesting. I think we used to bring to front in the past.
In that case, let's bringToFront right after OAuth.

Reply:
Done